normandy/recipes/static/actions/show-heartbeat/index.js

```diff
@@ -116,26 +116,28 @@ export default class ShowHeartbeatAction extends Action {
         let flow = new HeartbeatFlow(this);
         flow.save();
 
-        let extraTelemetryArgs = {
-            surveyId: surveyId,
-            surveyVersion: this.recipe.revision_id,
-        };
-        if (this.normandy.testing) {
-            extraTelemetryArgs.testing = 1;
-        }
-
+        // Object declared for showHeartbeat call below, so that optional
```

Comment:
+1 for adding comments, but this particular fact isn't necessary to point out. It's pretty clear why you're storing the data in a variable.